crates/python/qcs_sdk/qpu/rewrite_arithmetic.pyi

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
the tiniest of nits: extra newline at the top here.

Reply:
```suggestion
```